string[] object_id
geometry_msgs/PoseStamped[] object_poses
